src/test/java/com/google/devtools/build/lib/rules/objc/BazelJ2ObjcLibraryTest.java

```diff
@@ -690,6 +690,7 @@ protected Artifact j2objcArchive(String j2objcLibraryTarget, String javaTargetNa
 
   @Test
   public void testJ2ObjcInformationExportedFromObjcLibrary() throws Exception {
+    setBuildLanguageOptions("--incompatible_disable_objc_library_transition=false");
```

Comment:
flipping it back on these 2 was a cop out, it would be great if someone who knew the impact of j2objc changes here could review and recommend the right solution